gtfs/spec/en/reference.md

```diff
@@ -414,6 +416,23 @@ Trip planners normally calculate transfer points based on the relative proximity
 |  min_transfer_time | Optional | When a connection between routes requires an amount of time between arrival and departure (transfer_type=2), the **min_transfer_time** field defines the amount of time that must be available in an itinerary to permit a transfer between routes at these stops. The min_transfer_time must be sufficient to permit a typical rider to move between the two stops, including buffer time to allow for schedule variance on each route. |
 |   |  | The min_transfer_time value must be entered in seconds, and must be a non-negative integer. |
 
+### linked_datasets.txt
+
+File: **Optional**
+
+|  Field Name | Required | Details |
+|  ------ | ------ | ------ |
+|  url | **Required** | The **url** fields contains the URL to the linked dataset. The value must be a fully qualified URL that includes **http**:// or **https**://, and any special characters in the URL must be correctly escaped. See http://www.w3.org/Addressing/URL/4_URI_Recommentations.html for a description of how to fully qualified URL values. |
```

Comment:
We probably shouldn't limit to http and https - so it should read `The value must be a fully qualified URL that includes a scheme such as **http**:// or **https**://,...`.

I would also add `The value must be the exact URL a client could use to make a request that returns GTFS-realtime data in the protocol buffer format.

Reply:
Please include websocket as well?

Reply:
Agreed.

@barbeau & @skinkie: Do we want to defined exhaustively the protocols which are allowed (e.g. HTTP GET and WebSockets), or do we just want to just say "Should be a fully qualified URL" without specifying a protocol?

Reply:
I don't think we should exhaustively list protocols here - the purpose of this sentence and including "http/https" is just to give an example that we expect a fully qualified URL.  

To my knowledge all websocket implementations of GTFS-realtime are currently for `DIFFERENTIAL` feeds, which aren't currently officially supported in GTFS-realtime (see https://github.com/google/transit/issues/84), so I'd suggest omitting `ws://` examples to avoid confusion.  When `DIFFERENTIAL` feeds are officially supported we can add more text here that says something like "such as http for FULL_DATASET and ws for DIFFERENTIAL feeds".